Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Description: Drop building for windows

We don't support the crypto-cng backend on debian; it requires a
Windows installation to build that way.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -112,7 +112,6 @@
 default-features = false
 
 [features]
-crypto-cng = ["sequoia-openpgp/compression", "sequoia-openpgp/crypto-cng"]
 crypto-nettle = ["sequoia-openpgp/compression", "sequoia-openpgp/crypto-nettle"]
 default = ["sequoia-openpgp/default", "net"]
 net = ["sequoia-net", "tokio", "rand", "rand_distr"]
